### PR TITLE
Update presentations Slidev dependencies

### PR DESCRIPTION
**Pull request by runewake2**
This centralizes the Sli.dev dependency in the project and adds npm scripts for building and running each presentation.

This means you only need to download and install the package dependencies once.

In addition,

```
    "@slidev/theme-default": "*",
    "@slidev/theme-seriph": "*",
```

have been removed since they were not in use

### CODE REVIEW

File: presentations/README.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ To use the slides:
     npm install
```
Comment (jimbobbennett) [inline presentations/README.md:31]: The instruction above says 'Navigate to the relevant folder'. This should probably be changed to 'Navigate to the `presentations` folder' so it's clearer where the install should be run.

**Review by jimbobbennett — CHANGES_REQUESTED** [review]

File: presentations/README.md (hunk 1 above)
Comment (runewake2) [reply presentations/README.md:29]: Should be updated now

Comment (jimbobbennett) [reply presentations/README.md:29]: Do they need to run this in the `presentations` folder? If so we should specify this.

**Review by jimbobbennett — CHANGES_REQUESTED** [review]

**Comment by runewake2** [comment]
This README is already in that directory. You shouldn't need to change directories while reading this file.

**Comment by jimbobbennett** [comment]
> This README is already in that directory. You shouldn't need to change directories while reading this file.

Yup, just thinking about being explicit as the instructions say clone the repo, run the script.

**Comment by jimbobbennett** [comment]
Sorry - just realized there is another change needed. The npm install is run in the dev container. Can you also update the devcontainer.json to do the right thing!

File: presentations/README.md [hunk 2]
```diff
@@ -23,19 +23,17 @@ To use the slides:
 
 1. Clone this repo
 
-1. Navigate to the relevant folder
-
-1. Run the following command to install SliDev:
+1. From the `presentations` direction run the following command to install SliDev:
```
Comment (jimbobbennett) [inline presentations/README.md:26]:
```suggestion
1. From the `presentations` directory run the following command to install SliDev:
```

**Review by jimbobbennett — CHANGES_REQUESTED** [review]

**Comment by runewake2** [comment]
Updated the devcontainer

**Review by jimbobbennett — APPROVED** [review]

**Pull request merged**